### PR TITLE
Corrected example for how to use actions

### PR DESCRIPTION
**Pull request by daxmc99**

### CODE REVIEW

**Comment by dramich** [comment]
I think we should keep both versions and explain that using the client gets you retry logic for free.

**Comment by daxmc99** [comment]
Wouldn't using the previous method leave you open to encountering the race condition though?

**Comment by dramich** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -87,13 +87,24 @@ From the spec
 
 To perform the `setpodsecuritypolicytemplate` action on a project object these are the steps.
 
+This first method has built-in retry logic inside of `client.action()` when the error returned is 409
```
Comment (daxmc99) [inline README.md:90]: Added `when the error returned is 409`

File: README.md [hunk 2]
```diff
@@ -87,13 +87,24 @@ From the spec
...
+# perform the action via the client api
+api_client.action(obj=project, action_name="setpodsecuritypolicytemplate",
+                    podSecurityTemplateId=pspt.id)
+```
+Or alternatively, performing the action from the project context (which does not have built-in retry logic)
+```python
+
 project = api_client.create_project(name="test-project" clusterId="local")
-project.setpodsecuritypolicytemplate(podSecurityPolicyTemplateId="my-pspt")
+project.setpodsecuritypolicytemplat(podSecurityPolicyTemplateId="my-pspt")
```
Comment (daxmc99) [inline README.md]: *template*

**Pull request merged**